Header header

float32 s1
float32 s2
float32 s3
float32 s4
float32 s5
float32 s6
float32 s7
float32 s8

float32 s1d
float32 s2d
float32 s3d
float32 s4d
float32 s5d
float32 s6d
float32 s7d
float32 s8d

float32 e1
float32 e2
float32 e3
float32 e4
float32 e5
float32 e6
float32 e7
float32 e8


float32 J
float32 optNUM
float32 Z
float32 Tx
float32 Ty
float32 Tz
float32 Oz

float32 dtloop
float32 time

# float32[] transformed_features

# int32 first_min_index
# int32 second_min_index

# float32 s_bar_x
# float32 s_bar_y
# float32 s_bar_x_des
# float32 s_bar_y_des

# float32 transformed_sigma
# float32 transformed_sigma_square
# float32 transformed_sigma_square_log
# float32 sigma_des
# float32 sigma_square_des
# float32 sigma_log_des

# float32 angle_deg_des
# float32 angle_des_tan
# float32 transformed_tangent
# float32 transformed_angle_radian
# float32 transformed_angle_deg

